D2.5_1-5_Overview_DC_tech/D2.5_1-5_Overview_DC_tech.qmd

```diff
@@ -1,53 +1,66 @@
 
-The tools that are described here can be used for different tasks. 
-It can go from extraction of a data for a given spatial and temporal extent to the visualisation of the data, to the processing of the data in batches which also includes machine learning training. These different tasks have all different software needs.
+The tools that are described in this document can be used for different tasks. 
+These tasks range from extraction of a subset of data for a given spatial and temporal extent to the visualisation of the data, to the processing of the data in batches which also includes machine learning training. These different tasks have all different software needs.
 
 Another angle to look at is the size of the dataset. 
 The size of the data set can be described spatially, going from a local dataset like a single satellite acquisition to a regional dataset to a global harmonised dataset. 
 The size can also be in terms of the occupied data storage from a few megabytes to multiple petabytes.
 
-
 A classification of the type of the tool and the cloud readiness, describing, how well one can work on larger scale computations with this tool is shown in Figure 1. 
-Cloud readiness has different meanings for the different categories. 
-For data formats a higher cloud readiness indicates, that data can be accessed partially, so that a dataset which is provided in some storage system in the cloud does not need to be downloaded in full to access only a small subset of the data.
-For software the cloud readiness entails, that the software can semi automatically use the different computing resources, 
+We acknowledge that "Cloud readiness" is not a well-defined term and Cloud readiness can have different meanings for tools belonging to different categories. 
+For data formats a higher cloud readiness indicates, that cloud datasets data can be accessed partially, i.e. a dataset which is provided in an object store in the cloud does not need to be downloaded in full to access only a small subset of the data.
+For software the cloud readiness entails, that the software can semi automatically use the different computing resources in a scalable way, 
 which are provided in the cloud as a multi-computer cluster. 
 
-![Classification of the tools into cloud readiness](overview.svg)
+![*Overview of tools, their type and cloud-readiness* Be aware that there are some grey zones in this classification and many of the formats/tools at the lower end might be used in a cloud-context as well. However, we want to show the distinction between mature tools that started before the advent of cloud computing and newer tools that were designed specifically with cloud computing in mind.](overview.png)
```

Comment:
Why do you switch the format to .png?
Mature sounds as if the cloud ready tools are not mature. Should we talk about legacy tools?

Reply:
In some way I wanted to bring a positive annotation to older tools as well, maybe a different name would make sense...

Reply:
I switched to png because svg could not be rendered by my latex to render the pdf. We should use svg for html and png for the pdf